cmd/agent/install_script.sh

```diff
@@ -211,6 +209,31 @@ if [ -n "$DD_AGENT_FLAVOR" ]; then
     agent_flavor=$DD_AGENT_FLAVOR #Eg: datadog-iot-agent
 fi
 
+declare -A flavor_to_readable
+flavor_to_readable=(
+    ["datadog-iot-agent"]="Datadog IoT Agent"
+    ["datadog-dogstatsd"]="Datadog Dogstatsd"
+)
+nice_flavor=${flavor_to_readable[$agent_flavor]:-Datadog Agent}
```

Comment:
I would prefer to fail if `$agent_flavor` is not in `flavor_to_readable` (and the same thing everywhere else): if we add a new flavor things will silently not work

Reply:
For example, what about the Heroku Agent? What would be the behavior be in that case?

Reply:
Ugh, I totally forgot to handle this situation. Let me fix it.